### ISSUE TITLE
Error starting autocron on windows machine 

### ISSUE THREAD

**Issue by svhoy** [kbr/autocron#4]
I am trying to integrate autocron in a new django project. 

If starting the django server and start autocron with
``` 
class TournamentAutomationConfig(AppConfig):
	default_auto_field = "django.db.models.BigAutoField"
	name = "tournament_automation"

	def ready(self):
		autocron.start("db.sqlite3", 1)
```

or trying the admin on shell. 
I always get the error:
```
 File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\svenh\Documents\Programmieren\Privat\turnier_manager\tournament_django\.venv\Scripts\autocron.exe\__main__.py", line 5, in <module>
    from autocron.admin import main
  File "D:\svenh\Documents\Programmieren\Privat\turnier_manager\tournament_django\.venv\Lib\site-packages\autocron\__init__.py", line 25, in <module>
    _engine = Engine()
  File "D:\svenh\Documents\Programmieren\Privat\turnier_manager\tournament_django\.venv\Lib\site-packages\autocron\engine.py", line 47, in __init__
    signal.signal(signal.SIGCHLD, self._check_monitor_child)
                  ^^^^^^^^^^^^^^
AttributeError: module 'signal' has no attribute 'SIGCHLD'
```

After looking what I can do, it seems SIGCHLD doesn't work in windows. At the moment I can't provide a fix, but maybe there is a possbile workaround. 

Thanks for your help.

**Comment by kbr** [comment]
Yes, SIGCHLD is not available on windows. It was used to prevent zombies on unixoid systems (development is on macOS and most servers use some linux-distribution), but as the processes meanwhile check each other, it could be that handling this signal is not necessary any more. I have to go into this to be sure. Unfortunately there is no work around beside the "monkey patch" to comment out line 47 in engine.py locally on your installation. Anyway it should get fixed in a future release.

**Comment by kbr** [comment]
Should be fixed with version 1.2.2
